src/main/java/ca/corefacility/bioinformatics/irida/ria/web/users/UsersAjaxController.java

```diff
@@ -46,6 +43,28 @@ public TableResponse<UserDetailsModel> getUsersPagedList(@RequestBody AdminUsers
 		return UIUsersService.getUsersPagedList(request);
 	}
 
+	/**
+	 * Create a new user
+	 *
+	 * @param userCreateRequest a {@link UserCreateRequest} containing details about a specific user
+	 * @param principal         a reference to the logged in user
+	 * @param locale            the logged in user's request locale
+	 * @return A list or errors
+	 */
+	@RequestMapping(value = "/create", method = RequestMethod.POST)
+	public ResponseEntity<Map<String, String>> createUser(@RequestBody UserCreateRequest userCreateRequest,
+			Principal principal, Locale locale) {
+
+		UserDetailsResponse response = UIUsersService.createUser(userCreateRequest, principal, locale);
+
+		if (response.hasErrors())
+			return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(response.getErrors());
+			//TODO: handle mail failure?
```

Comment:
Are you going to handle this?

Reply:
I was, but then I started thinking. What would the users recourse be? Assuming the user is successfully created.